js/bootstrap-slider.js

```diff
@@ -893,6 +902,15 @@
 				}
 				this.eventToCallbackMap[evt].push(callback);
 			},
+            _unbindNonQueryEventHandler: function(evt, callback) {
+                if(this.eventToCallbackMap[evt]!==undefined) {
+                    for (var i = 0; i < this.eventToCallbackMap[evt].length(); i++) {
+                        if (this.eventToCallbackMap[evt][i] == callback) {
+                            this.eventToCallbackMap[evt].splice(i, 1);
```

Comment:
i would add a `break;` statement after this line, which will prevent the code from looping through all of the event to callback mappings;